cmd/lk/app.go

```diff
@@ -291,13 +309,27 @@ func setupTemplate(ctx context.Context, cmd *cli.Command) error {
 		return err
 	}
 
+	tf, err := bootstrap.ParseTaskfile(appName)
+	if err != nil {
+		return err
+	}
+
 	fmt.Println("Instantiating environment...")
 	addlEnv := &map[string]string{"LIVEKIT_SANDBOX_ID": sandboxID}
-	env, err := instantiateEnv(ctx, cmd, appName, addlEnv)
+	envOutputFile := ".env.local"
+	if customOutput, ok := tf.Vars.Get("env_file").Value.(string); ok {
+		envOutputFile = customOutput
+	}
+	envExampleFile := ".env.example"
+	if customExample, ok := tf.Vars.Get("env_example").Value.(string); ok {
+		envExampleFile = customExample
```

Comment:
Despite what it says here https://github.com/livekit-examples/index?tab=readme-ov-file#creating-your-own-template, a `taskfile.yaml` is optional and `bootstrap.ParseTaskfile` wil return `nil, nil` if none is present. I didn't check the Task docs, but just wanted to be sure there isn't a NPE waiting in this code!

Reply:
makes sense, fixed that and tested it too